### PR TITLE
[SDA-5894] fix: adding trim spaces and tabs when validating cluster name

### PR DESCRIPTION
**Pull request by gdbranco**
Related issue: https://issues.redhat.com/browse/SDA-5894
# What
Interactive mode of input is still not trimming 'invisible characters'

# Why
Facilitates user input

### CODE REVIEW

File: pkg/ocm/helpers.go [hunk 1]
```diff
@@ -83,6 +83,7 @@ func IsValidClusterName(clusterName string) bool {
 
 func ClusterNameValidator(name interface{}) error {
 	if str, ok := name.(string); ok {
+		str := strings.Trim(str, " \t")
```
Comment (gdbranco) [inline pkg/ocm/helpers.go:86]: Ended up doing in this manner, I looked into the survey dependency and they have a TransformString option which would have been useful, but it only applies transform after validation, so we would either have to allow ' \t' on the regex or trim the internal string anyway

Comment (gdbranco) [reply pkg/ocm/helpers.go:86]: @vkareh PTAL

Comment (zgalor) [reply pkg/ocm/helpers.go:86]: strings.Trim is fine

**Pull request merged**